plugins/by-name/fidget/default.nix

```
@@ -33,649 +27,460 @@ let
-              suppress_on_insert = suppressOnInsert;
-              ignore_done_already = ignoreDoneAlready;
-              ignore_empty_message = ignoreEmptyMessage;
-              notification_group = notificationGroup;
-              clear_on_detach = clearOnDetach;
-              inherit ignore;
-              display = with display; {
-                render_limit = renderLimit;
-                done_ttl = doneTtl;
-                done_icon = doneIcon;
-                done_style = doneStyle;
-                progress_ttl = progressTtl;
-                progress_icon = progressIcon;
-                progress_style = progressStyle;
-                group_style = groupStyle;
-                icon_style = iconStyle;
-                inherit priority;
-                skip_history = skipHistory;
-                format_message = formatMessage;
-                format_annote = formatAnnote;
-                format_group_name = formatGroupName;
-                overrides = helpers.ifNonNull' overrides (mapAttrs (_: processNotificationConfig) overrides);
-              };
-              lsp = with lsp; {
-                progress_ringbuf_size = progressRingbufSize;
-              };
-            };
-            notification = with notification; {
-              poll_rate = pollRate;
-              inherit filter;
-              history_size = historySize;
-              override_vim_notify = overrideVimNotify;
-              configs = helpers.ifNonNull' configs (
-                mapAttrs (
-                  _: value: if isString value then helpers.mkRaw value else processNotificationConfig value
-                ) configs
-              );
-              inherit redirect;
-              view = with view; {
-                stack_upwards = stackUpwards;
-                icon_separator = iconSeparator;
-                group_separator = groupSeparator;
-                group_separator_hl = groupSeparatorHl;
-              };
-              window = with window; {
-                normal_hl = normalHl;
-                inherit winblend border;
-                border_hl = borderHl;
-                inherit zindex;
-                max_width = maxWidth;
-                max_height = maxHeight;
-                x_padding = xPadding;
-                y_padding = yPadding;
-                inherit align relative;
-              };
-            };
-            integration = with integration; {
-              nvim-tree = with nvim-tree; {
-                inherit enable;
-              };
-            };
-            logger = with logger; {
-              inherit level;
-              float_precision = floatPrecision;
-              inherit path;
-            };
-          }
-          // cfg.extraOptions;
-      in
-      ''
-        require("fidget").setup${lib.nixvim.toLuaObject setupOptions}
-      '';
+        [
+          "You have set `plugins.fidget.settings.integrations.nvim-tree.enable` to true but have not enabled `plugins.nvim-tree`."
```

```suggestion
          "Nixvim(plugins.fidget): You have set `plugins.fidget.settings.integrations.nvim-tree.enable` to true but have not enabled `plugins.nvim-tree`."
```

Is what we've been moving towards, but not a big deal since we can handle in a treewide.

---

Updated. Needs to queue the merge again.